packages/react/src/components/Authenticator/hooks/useFormHandlers/useFormHandlers.ts

```diff
@@ -3,7 +3,7 @@ import { useCallback } from 'react';
 import { getFormDataFromEvent } from '../../../../helpers/utils';
 import { useAuthenticator } from '../useAuthenticator';
 
-export default function useFormHandlers() {
+export function useFormHandlers() {
```

Comment:
Curious why we need to update this from a `default` export?

Reply:
Just for consistency, the other hooks are using named export. We prefer named export because it allows us to shared the exact same name.